cultivation/scripts/flashcore/exceptions.py

```
@@ -0,0 +1,40 @@
+from typing import Optional
+
+class DatabaseError(Exception):
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)